.github/workflows/CI.yml

```diff
@@ -97,6 +97,8 @@ jobs:
     strategy:
       matrix:
         include:
+          - namespace: application-loadbalancers
+            keywords: "application_load_balancer, target_group"
```

Comment:
I would add here maybe `application-load-balancer` too, to match the test file names

Reply:
Same for `target-group`